toolkit/Makefile

```diff
@@ -120,39 +120,10 @@ ifeq ($(USE_PREVIEW_REPO),y)
    endif
 endif
 
-ifneq ($(strip $(SRPM_PACK_LIST)),)
-LOCAL_SPECS = $(wildcard $(addprefix $(SPECS_DIR)/*/,$(addsuffix .spec,$(strip SRPM_PACK_LIST))))
-else # Empty pack list, build all under $(SPECS_DIR)
-LOCAL_SPECS = $(shell find $(SPECS_DIR)/ -type f -name '*.spec')
-endif
-
-LOCAL_SPEC_DIRS = $(foreach spec,$(LOCAL_SPECS),$(dir $(spec)))
-
 CA_CERT     ?=
 TLS_CERT    ?=
 TLS_KEY     ?=
 
-# Build defines
-DIST_TAG           ?= .cm2
-# Running 'git' as the owner of the repo, so it doesn't complain about the repo not belonging to root.
-BUILD_NUMBER       ?= $(shell runuser -u $$(stat -c "%U" $(PROJECT_ROOT)) -- git rev-parse --short HEAD)
-# an empty BUILD_NUMBER breaks the build later on
-ifeq ($(BUILD_NUMBER),)
-   BUILD_NUMBER = non-git
-endif
-RELEASE_MAJOR_ID   ?= 2.0
-# use minor ID defined in file (if exist) otherwise define it
-# note this file must be single line
-ifneq ($(wildcard $(OUT_DIR)/version-minor-id.config),)
-   RELEASE_MINOR_ID ?= .$(shell cat $(OUT_DIR)/version-minor-id.config)
-else
-   RELEASE_MINOR_ID ?= .$(shell date +'%Y%m%d.%H%M')
-endif
-RELEASE_VERSION    ?= $(RELEASE_MAJOR_ID)$(RELEASE_MINOR_ID)
-
-# Image tag - empty by default. Does not apply to the initrd.
-IMAGE_TAG          ?=
-
```

Comment:
I'd be careful about moving these variables around, as we already have some scripts depending on them being here. From the ones I was able to find so far:

1. This repo:

- `toolkit/scripts/rpmops.sh`

2. Our pipelines: look for `grep.*Makefile` across all files.

It seems like we're currently extracting `DIST_TAG` and `RELEASE_MAJOR_ID`. The problem in my opinion is on the side of how these variables are extracted, so if we can add `make` targets, which return these values, are quick and don't produce any extra garbage, then that would be the ideal way forward.

Reply:
Will do, I'll tinker around and see what I can figure out.

Reply:
I think we have a fix for that now in our pipelines, unblocking on this comment.